projects/packages/my-jetpack/_inc/components/product-cards-section/connected-product-card.jsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import React, { useCallback } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import ProductCard from '../product-card';
+import { useProduct } from '../../hooks/use-product';
+import { AntiSpamIcon } from '../icons';
+import useMyJetpackNavigate from '../../hooks/use-my-jetpack-navigate';
+
+const ConnectedProductCard = ( { admin, slug, showDeactivate } ) => {
+	const { detail, status, activate, deactivate, isFetching } = useProduct( slug );
+	const { name, description, manageUrl } = detail;
+
+	const navigateToConnectionPage = useMyJetpackNavigate( '/connection' );
+	const navigateToAddProductPage = useMyJetpackNavigate( `add-${ slug }` );
+
+	/*
+	 * Redirect to manage URL
+	 */
+	const onManage = useCallback( () => {
+		window.location = manageUrl;
+	}, [ manageUrl ] );
+
+	return (
+		<ProductCard
+			name={ name }
+			description={ description }
+			status={ status }
+			icon={ <AntiSpamIcon /> }
```

Comment:
We should receive the icon, or map it based on the slug (I prefer the first one, but is up to you), this way we'll show `AntiSpamIcon` in all cards.

Reply:
fixing...

Reply:
before | after
----|-----
<img width="870" alt="image" src="https://user-images.githubusercontent.com/77539/156795734-ec4aa957-0114-4b3b-a513-0745dd1e0433.png"> | <img width="869" alt="image" src="https://user-images.githubusercontent.com/77539/156795942-76f060d7-748a-457b-a113-584a84e69c8d.png">

> We should receive the icon, or map it based on the slug (I prefer the first one, but is up to you), 

well... the spirit of this new _connected_ component is to resolve these handling magically, in the opposite direction than the ProductCard component.

[Addressed here](https://github.com/Automattic/jetpack/pull/23224/commits/b8ab3bf9a7b5e19c1730230f1d5a1ed0cf7b241d)